Cookbook/Technical-Documents/Localizations.md

```diff
@@ -0,0 +1,132 @@
+
+![Select the app target, Build Settings, User Defined](Assets/Localizations-appLanguages-buildSettings-userDefined.png)
+
+### _Excluded Source File Names_ and _Included Source File Names_ under `Build option`
+In the _Excluded Source File Names_ we exclude any language (`.lproj/`) previously set:
+`*.lproj/*` 
+
+In the _Included Source File Names_ we include the languages (`.lproj/`) set in the above mentioned `app_languages` array:
+`${APP_LANGUAGES}` in Babylon US for example it is `en-US.lproj/*` `es-US.lproj/*`.
+
+## How to add a new or changing an existing key/value
+### Add a new key/value
+In Lokalise, select the corresponding project. 
+Click on the `Add key ⌘K` button
+- `key` - Give a name for the key following the bread-crumb style. For example: `add_family_member_email_placeholder`, `biometric_touchid_primer_description`.
+    
+- `Base language value`: The actual string value corresponding to the first language in the list. Placeholders are supported with the `%@` where dynamic values are expected. 
+
+Note: the translations will be done by a third party that has access to lokalise and it should be done and notified by your squad.
+
+### Edit a key/value
+Search and select the key you'd like to update and change any language you need. The languages you may not have the values will be marked as `Not-verified` and the translators will take care.
+
+## How to add a new Language/Locale
+
+### 1. Xcode: create Files if needed
+- Select the Babylon Project / Info / Localizations then click on the `+` button and choose the Language/Locale you want to add.
+![Create file, if needed](Assets/Localizations-createFile.png)
+
+- Select the `.strings` file in the Project Navigator, review it in the File Inspector / Localization and select the language/locale you want.
+- In `Fastlane/Lanes/lokalise`, add the language/locale code (`en_US` for example) in the `langs` parameter for the desired target. `langs: 'es_US,en_US'` (**comma separated without space**). 
+
+### 2. Lokalise.co
+- If the project still doesn't exist, click on the `+` plus button beside other flags. Find the desired language on the list, and then add it.
+Note: someone else from your squad should have done this before.
+
+#### 2.1 Populate the files - pull from lokalise
+- Check [Lokalise pull guide](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/Lokalise.md).
+
+#### 2.2 Commit just your changes
+Stage just your changes (additions and editions) on git. Don't stage any additions nor change that you don't recognize. It's really nice to be proactive but not in this case. It might cause unexpected/premature changes. Yes, discard others's changes.
+
+## Target specific localizable - Lokalise project and `strings` files
+
+As the name suggests, it's a `.strings` file that contains texts intended to be present **only** in a specific app target.
+The values in this file will be priority over the main file value.
+<br><br>
```

Comment:
`<br>` tags should be avoided in markdown ([ref](https://github.com/google/styleguide/blob/gh-pages/docguide/style.md#trailing-whitespace))

```suggestion

```